packages/account/src/Sections/Security/SelfExclusion/self-exclusion.tsx

```diff
@@ -384,7 +393,7 @@ const SelfExclusion = observer(({ is_app_settings, overlay_ref, setIsOverlayShow
     const getLimits = () => {
         setState({ is_loading: true });
 
-        WS.authorized.getLimits({ get_limits: 1 }).then((limits: FormikValues) => {
+        getLimitsFromStore().then((limits: FormikValues) => {
```

Comment:
how can we make sure store value is the latest response?

Reply:
@amina-deriv this is the code of `getLimits` function from the store, you can see we make the same request

<img width="611" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/d45e36b2-a11b-4e12-a68b-39e398fa2dba">

Reply:
alright. So we are making new request  The name confused me